test/dicon/executor_test.exs

```diff
@@ -45,4 +48,15 @@ defmodule Dicon.ExecutorTest do
     message = "(in Dicon.ExecutorTest.FakeExecutor) copy failed"
     assert_raise Mix.Error, message, fn -> Executor.copy(conn, :fail, :fail) end
   end
+
+  test "write_file/4" do
+    conn = Executor.connect(:whatever)
+
+    assert Executor.write_file(conn, :target, "content") == :ok
```

Comment:
The spec for `write_file/4` says the target should be `char_data`, maybe we should respect that here?